scripts/bin/update-checksum

```diff
@@ -57,29 +57,42 @@ for package in "${@}"; do
 
 	# Calculate new checksum in subshell.
 	echo
-	new_checksum=$(
+	new_checksum=($(
 		source "${buildsh_path}" 2>/dev/null
+		for url in ${TERMUX_PKG_SRCURL[@]}; do
+			dl_tmpdir=$(mktemp -d "${TMPDIR-/tmp}/termux.src.dl.XXXXXXXX")
 
-		dl_tmpdir=$(mktemp -d "${TMPDIR-/tmp}/termux.src.dl.XXXXXXXX")
+			if [ -n "${dl_tmpdir}" ] && [ -d "${dl_tmpdir}" ]; then
+				if ! curl --fail --location --retry 3 --output "${dl_tmpdir}/source-bundle" \
```

Comment:
<sub>(Nitpick, feel free to ignore)</sub>
Not a big fan of this being split here.
```suggestion
				if ! curl --fail --location --retry 3 --output "${dl_tmpdir}/source-bundle" "$url"; then
```
Either keep it on 1 line, or split it after `--retry 3`.